src/app/cakes/cake.class.ts

```diff
@@ -0,0 +1,30 @@
+export const enum CakeStatus {
+    Available = "Available",
+    LastUnits = 'LastUnits',
+    OutOfStock = 'OutOfStock'
+};
+
+export interface ICake {
```

Comment:
Since we are using a Single File Responsibility file distribution, this Interface should live in a separate file named `cake.interface.ts`.